### PR TITLE
Add the InMigration sync state

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Only used during migration.

PR created here to speed up review.

### CODE REVIEW

File: AnkiDroid/src/main/res/values/01-core.xml [hunk 1]
```diff
@@ -155,6 +155,7 @@
     <string name="sd_card_not_mounted">Device storage not mounted</string>
     <string name="empty_cloze_warning">Edit this note and add some cloze deletions. (%1$s)</string>
     <string name="button_sync" comment="Text of the sync button">Sync</string>
+    <string name="button_sync_disabled" comment="Text of the sync button when disabled">Disabled sync</string>
```
Comment (david-allison) [inline AnkiDroid/src/main/res/values/01-core.xml:158]:
```suggestion
    <string name="button_sync_disabled" comment="Text of the sync button when disabled">Sync temporarily disabled</string>
```

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
one nit on string. Optional, but should be tested to ensure the string isn't truncated

**Comment by Arthur-Milchior** [comment]
2022-10-13-180028_236x85_scrot
On emulator, it fits

**Review by mikehardy — APPROVED** [review]
I feel like this is fine but also that I just approved something that centralized sync icon titling? I will leave it with you to modify this + merge if you need to move the menutItem.setTitle call for SyncIconState.Disabled, it looks fine otherwise

**Review by mikehardy — APPROVED** [review]
Still LGTM - thanks!

**Comment by mikehardy** [comment]
macos unrelated flake

```

ReviewerNoParamTest > drawerConflictsIfUp FAILED
    java.lang.AssertionError: 
    Expected: <true>
         but: was <false>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
        at com.ichi2.anki.ReviewerNoParamTest.drawerConflictsIfUp(ReviewerNoParamTest.kt:216)
```

**Pull request merged**